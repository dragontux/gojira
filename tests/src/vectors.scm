(display #(a b c 1 2 3 :foo :bar (baz :que)))
(newline)

(display (vector? #(lorem ipsum)))
(newline)

(display (vector? '(this is probably not a vector)))
(newline)

(display ((lambda (foo)
   (display (vector-ref foo 0))
   (newline)

   (display (vector-ref foo 2))
   (newline)

   (display (+ (vector-ref foo 1) 10))
   (newline)
   
   foo)
 #(:foo 123 wut)))
(newline)
