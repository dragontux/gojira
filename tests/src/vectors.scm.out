#(a b c 1 2 3 :foo :bar (baz :que))
#t
#f
:foo
wut
133
#(:foo 123 wut)
